pkg/virt-controller/watch/migration.go

```diff
@@ -1011,6 +1033,35 @@ func (c *MigrationController) deletePod(obj interface{}) {
 	c.enqueueMigration(migration)
 }
 
+func (c *MigrationController) updatePDB(old, cur interface{}) {
+	curPDB := cur.(*v1beta1.PodDisruptionBudget)
+	oldPDB := old.(*v1beta1.PodDisruptionBudget)
+	if curPDB.ResourceVersion == oldPDB.ResourceVersion {
+		return
+	}
+
+	// Only process PDBs manipulated by this controller
+	migrationName := curPDB.Labels[virtv1.MigrationNameLabel]
+	if migrationName == "" {
+		return
+	}
+
+	objs, err := c.migrationInformer.GetIndexer().ByIndex(cache.NamespaceIndex, curPDB.Namespace)
+	if err != nil {
+		return
+	}
+
+	for _, obj := range objs {
+		vmim := obj.(*virtv1.VirtualMachineInstanceMigration)
+
+		if vmim.Name == migrationName {
+			log.Log.V(4).Object(curPDB).Infof("PDB updated")
+			c.enqueueMigration(vmim)
+			return
```

Comment:
There shouldn't be (assuming we have no bugs), not sure if we should return early here to (not really) enforce this, probably it's better to remove the return to avoid confusion but in normal conditions, 1 PDB is associated with 1 VMI at all times.